documents/minutes/2020-07-19 WCA Board Meeting.md

```diff
@@ -0,0 +1,46 @@
+<div class="text-center">
+MINUTES OF MEETING OF THE BOARD OF DIRECTORS OF WORLD CUBE ASSOCIATION
+
+A California Nonprofit Public Benefit Corporation
+</div>
+
+On July 19, 2020 at 7:00 p.m. UTC, the Board of Directors of World Cube Association (the “Board”) met for a regular meeting via Google Hangouts. The following directors were present at the meeting:
+
+<div class="text-center">
+Ethan Pride <br>
+Henrik Buus Aagaard <br>
+Robert Burton <br>
+</div>
+
+
+<b class="text-center">FILLING VACANCIES WITHIN THE BOARD</b>
+
+The Board discussed filling vacancies within the Board as required by section 4.1 of the WCA Bylaws. The Board noted that at the expiration of the term of Christopher Wright there shall be two vacancies within the Board. The process of an election has begun to fill these vacancies.
+
+<b class="text-center">COMPETITION SAFETY POLICY REVIEW</b>
+
+The Board discussed a review of the WCA Competition Safety Policy and concluded that no updates are required at this time. The Board shall review the policy for any amendments in one fortnight.
+
+<b class="text-center">SELECTION OF WCA CHAMPIONSHIPS</b>
+
+RESOLVED, that Almere, Netherlands shall be the host city of the WCA World Championship 2021 on July 22-25 2021.
+
+RESOLVED, that Semey, Kazakhstan shall be the host city of the WCA Asian Championship 2022 on July 6-9 2022
```

Comment:
I think a dot is missing here and in the next two lines.

Reply:
Thanks for spotting that!